### ISSUE TITLE
Character Sections (Rune Magic, Sorcery ETC) and disappearing skills.

### ISSUE THREAD

**Issue by MashiraTimire** [sun-dragon-cult/fvtt-system-rqg#736]
It is very frustrating to get into tabs on the character sheet. You have to grab in an item for that tab for it to open. I'd prefer there to be something to click rather than needing to drag in an item, and a [+] symbol to make a new skill, new sorcery, new rune, etc.

I know Sorcery is getting tweaked later, but at the moment I have to drag in a skill like Alchemy, then turn it into a Sorcery and edit it. But that means I cannot drag in Alchemy again.

Furthermore, if a skill is 'unhooked' from its skill category (agility, manipulation, etc), it vanishes and you can't select it to re-assign it to a category, and cannot drag in the same skill as it's 'already in the sheet' and you can't have duplicates.

**Comment by wake42** [comment]
The idea is that one actor type should be able to handle all types of actors. To achieve that everyting that does not yet "exist" is hidden to start with. That means that a ghost for example will look more sparse compared to a normal PC actor. It also means that I can not have a `+` button to create the first instance of an item that would be hidden behind a tab.

I have added `+`buttons to items that I think the players themselves should be able to add (Passions & Gear). Creating a weapon or armor from scratch is too complicated so there is no `+` button there. That said, there of course is a grey zone - should a player be able to add a skill on their own? I think not and that the GM should do that, but it's of course debatable.

The way I envision building a character is that you start from a wiki actor like `Human (Base)` and drag whatever extra you need from the wiki compendium. This case was about sorcery, and there are nothing for that - but if you use sorcery a lot, I would suggest creating a compendium with the sorcery spells you think you will need to have something to drag from.

Your other question/bugreport is about Sorcery skills and that you can "loose" a skill if you add a rune but have the category set to something else than "Magic". I agree that that's a bug, I thought I had a "catch" to find lost skills, but it is apparently not working. Until Sorcery is developed for real I should probably just don't look at the category for determining if a skill actually is a sorcery spell, and interpret any skill with runes as a sorcery skill.

**Comment by MashiraTimire** [comment]
Right, so it seems more that the difficulty to open the Sorcery Tab at times is down to it not being fully developed yet, so I can see that.

For the losing of skills, couldn't you have a header akin to "Communication" and "Manipulation" called "Uncategorised"? That'd just be an easy catch for anything missing.

And I think a + button for added skills would be useful for fringe cases. Unless you have particularly unruly players I don't see the issue with it.

**Comment by wake42** [comment]
I see your point and will have another look at what items a player should be able to add themselves.
